ADD Post FontAwesome and Post react init

cristofer-dev opened the pull request:
- Add post how add FontAwesome in ReactAPP
- Add post react init general react app

 Changes to be committed:
	new file:   src/assets/img/fontawesome.svg
	new file:   src/assets/img/react.svg
	renamed:    src/posts/2019-08-24-tools.md -> src/posts/2018-08-24-tools.md
	renamed:    src/posts/2020-09-08-autorizador-apigateway.md -> src/posts/2018-09-08-autorizador-apigateway.md
	new file:   src/posts/2018-09-23-fontawesome.md
	new file:   src/posts/2018-09-27-react-init.md

cristofer-dev committed:
ADD Post FontAwesome and Post react init

 Changes to be committed:
	new file:   src/assets/img/fontawesome.svg
	new file:   src/assets/img/react.svg
	renamed:    src/posts/2019-08-24-tools.md -> src/posts/2018-08-24-tools.md
	renamed:    src/posts/2020-09-08-autorizador-apigateway.md -> src/posts/2018-09-08-autorizador-apigateway.md
	new file:   src/posts/2018-09-23-fontawesome.md
	new file:   src/posts/2018-09-27-react-init.md

cristofer-dev committed:
UPDATE post tooling

Changes to be committed:
	modified:   src/posts/2018-08-24-tools.md

Xabadu commented on src/posts/2018-08-24-tools.md:
```diff
@@ -9,11 +9,11 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
-- **lint-staged**: Corre `eslint` a los archivos que estan en `stage` antes del commit.
+* **eslint**: un linter para JavaScript.
+* **airbnb style**: guía de estilo para JavaScript. ( usadas por el lint )
+* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
```
`automática`

Xabadu commented on src/posts/2018-08-24-tools.md:
```diff
@@ -9,11 +9,11 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
+* **eslint**: un linter para JavaScript.
+* **airbnb style**: guía de estilo para JavaScript. ( usadas por el lint )
+* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
+* **husky**: Nos permite definir acciones que seran ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateara nuestro código aplicando las reglas de airbnb.
```
`serán ejecutadas`
`ejecución`
`formateará`

Xabadu commented on src/posts/2018-08-24-tools.md:
````diff
@@ -55,14 +55,41 @@ Le decimos a eslint que reglas aplicar y con quien debe integrarse, creando `.es
 }
 ```
 
+Tambien podemos sobre-escribir usando prettier definiendo reglas en el archivo `.prettierrc` en la raiz del proyecto.
````
`También`
`raíz`

Xabadu commented on src/posts/2018-09-27-react-init.md:
```diff
@@ -0,0 +1,56 @@
+
+- i18next
+- Linter
+- fontAwwesome
```
`Awesome`

Xabadu commented on src/posts/2018-09-27-react-init.md:
```diff
@@ -0,0 +1,56 @@
+- i18next
+- Linter
+- fontAwwesome
+- Graficos
```
`Gráficos`

cristofer-dev committed:
FIX grammar

 Changes to be committed:
	modified:   src/posts/2018-08-24-tools.md
	modified:   src/posts/2018-09-27-react-init.md

cristofer-dev commented:
Done @Xabadu ! :)
Thanks!

Xabadu commented on src/posts/2018-08-24-tools.md:
```diff
@@ -11,8 +11,8 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
-* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
-* **husky**: Nos permite definir acciones que seran ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateara nuestro código aplicando las reglas de airbnb.
+* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automática.
+* **husky**: Nos permite definir acciones que serán ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateará nuestro código aplicando las reglas de airbnb.
```
`ejecutadas`

cristofer-dev committed:
FIX one word  :/

 Changes to be committed:
	modified:   src/posts/2018-08-24-tools.md

cristofer-dev committed:
UPDATE post React Init

 Changes to be committed:
	modified:   src/posts/2018-09-27-react-init.md

cristofer-dev committed:
Update Post tooling and React

Changes to be committed:
	modified:   src/posts/2018-09-23-fontawesome.md
	renamed:    src/posts/2021-09-22-i18next-react.md -> src/posts/2022-09-22-i18next-react.md

cristofer-dev commented:
Thanks @Xabadu !
